shell.core/src/main/java/org/metaborg/spoofax/shell/client/Repl.java

```diff
@@ -47,6 +37,7 @@
         this.display = display;
         this.invoker = invoker;
 
+        // TODO: this does not really belong here.
```

Comment:
@Balletie Where else would you move these?

@gfokkema Why should the prompt be the same if the complete user interface is different? I think that every implementation of `IEditor` and `IDisplay` knows best which environment/user interface it is in and is therefore best knowledgeable to decide its, for example, prompt.